pkg/cvo/cvo.go

```diff
@@ -652,3 +661,28 @@ func (optr *Operator) defaultPreconditionChecks() precondition.List {
 		preconditioncv.NewUpgradeable(optr.cvLister),
 	}
 }
+
+func (optr *Operator) getHTTPClientTransport() (http.RoundTripper, error) {
+	proxyURL, cmNameRef, err := optr.getHTTPSProxyURL()
```

Comment:
https://github.com/openshift/cluster-version-operator/blob/59601a89bb7a65cf2fbaa4697d5142091f68534f/pkg/cvo/availableupdates.go#L44-L53

does similar, so maybe both can use common method.

Reply:
This is now centralized in `getTransportOpts`.